internal/datasourcev2/polling_data_source.go

```diff
@@ -93,6 +91,8 @@ func (pp *PollingProcessor) Fetch(_ context.Context) (*subsystems.Basis, error)
 func (pp *PollingProcessor) Sync(closeWhenReady chan<- struct{}) {
 	pp.loggers.Infof("Starting LaunchDarkly polling with interval: %+v", pp.pollInterval)
 
+	ctx := context.TODO()
```

Comment:
Maybe we should comment this, as I assume it isn't the ideal.

Reply:
Done.